cloudnet-driver/src/main/java/de/dytanic/cloudnet/driver/service/ProcessConfiguration.java

```diff
@@ -17,11 +18,18 @@
     protected int maxHeapMemorySize = -1;
 
     protected Collection<String> jvmOptions;
+    protected Collection<String> processParameters = new ArrayList<>();
```

Comment:
This initialization is redunant

Reply:
![Screenshot_2](https://user-images.githubusercontent.com/54715954/87327707-b393ca80-c534-11ea-8bbb-bbc7327e6f7b.png)

Reply:
But this does not change the facts

Reply:
I will check if my Null Check is the only thing required ^^
Or shouldn't I include methods for auto updating?